### DISCUSSION TITLE
Sigma.js & Zustand

### DISCUSSION THREAD

**Discussion by cchin25** [jacomyal/sigma.js#1494] [Q&A] [answered]
This is probably way too specific of use-case, but was curious if anyone's implemented Sigma w/ Zustand as their state management solution in React? I'm familiar with the react-sigma project, but I'd prefer not having an additional upstream dependency, and it appears like they haven't updated in months. 

I'd also prefer to work directly with the Sigma/Graphology APIs. TIA

**Comment by jacomyal** [comment, marked as answer]
I can't speak about using sigma with Zustand unfortunately, but I can say that react-sigma is not that active just because it is stable enough. @sim51, the maintainer, is also a maintainer of sigma, and is quite reactive when there are bugs to fix. Also, at @ouestware, we use react-sigma in almost every React-based projects we have (about 10 different projects at least), and I only go with a custom component when I have need very specific lifecycles.

**Reply by cchin25** [reply]
I appreciate the insight, since that's the case I'll definitely take another look at it, certainly would make life easier. Thanks!
